frontend/app/generateGameInterface.js

```diff
@@ -71,6 +71,11 @@ export function endTurn(JSONSend) {
         serverAddress: 'cgi-bin/increment_turn.py',
         jsonObject: {player_id: id},
     });
+    // Calls page to checking if player declared bankrupt
+    JSONSend({
+        serverAddress: 'cgi-bin/is_player_bankrupt.py',
+        jsonObject: {player_id: id},
+    });
```

Comment:
Rather than sending a new request, could you call `is_bankrupt()` on the server side from `increment_turn.py`? It would save some unnecessary network communication if we could do it!

Reply:
I had left it like that for clarity purposes initially but it does make more sense and save unneeded back and forth communication. I'll change that.